test/unit/whitehall/publishing_api_test.rb

```diff
@@ -22,6 +28,42 @@ class Whitehall::PublishingApiTest < ActiveSupport::TestCase
     assert_requested request
   end
 
+  test "#publish sends unpublishing for case studies to the content store" do
+    edition = create(:draft_case_study)
+    unpublishing = create(:unpublishing, edition: edition)
+
+    payload = PublishingApiPresenters::Unpublishing.new(unpublishing).as_json
+    request = stub_publishing_api_put_item(unpublishing.document_path, payload)
+
+    Whitehall::PublishingApi.publish_async(unpublishing)
+
+    assert_requested request
+  end
+
+  test "#publish skips sending unpublishings for formats other than case study" do
+    edition = create(:draft_publication)
+    unpublishing = create(:unpublishing, edition: edition)
+
+    Whitehall::PublishingApi.publish_async(unpublishing)
+  end
```

Comment:
I realise you are relying on Webmock to complain if a request is made that has not been stubbed, but it still seems a bit odd without any kind of assertion. Should we explicitly assert that put_item has not been called, here and in 64 belowe?